test/conformance/source/environment.cpp

```diff
@@ -199,6 +201,14 @@ DevicesEnvironment::DevicesEnvironment(int argc, char **argv)
         error = "Could not find any devices associated with the platform";
         return;
     }
+    // Get an environment variable (NUMBER_OF_DEVICES) to run tests on the appropriate number of devices
```

Comment:
what does `appropriate` mean here? 😄 perhaps `wanted`...?

Reply:
I've changed it.